robot/Cumulus/tests/browser/gift_entry/change_labels_tmpbldr.robot

```diff
@@ -0,0 +1,92 @@
+    Click Gift Entry Button                 Next: Batch Settings
+    Add Batch Table Columns                 Campaign                        Check Number          Record Type
+    Click Gift Entry Button                 Save & Close
+    Current Page Should Be                  Landing                         GE_Gift_Entry
+    Click Link                              Templates
+    Wait Until Page Contains                ${TEMPLATE}
+    Store Template Record Id                ${TEMPLATE}
+    #Verify new field labels are displayed on newly created batch with new template
+    Click Gift Entry Button                 New Batch
+    Wait Until Modal Is Open
+    Select Template                         ${TEMPLATE}
+    Load Page Object                        Form                            Gift Entry
+    Fill Gift Entry Form
+    ...                                     Batch Name=${CAMPAIGN_REC}[Name] Automation Batch
+    ...                                     Batch Description=This is a test batch created via automation script
+    Click Gift Entry Button                 Next
+    Click Gift Entry Button                 Save
+    Current Page Should Be                  Form                            Gift Entry            title=Gift Entry Form
+    ${batch_id} =                           Save Current Record ID For Deletion     ${NS}DataImportBatch__c
+    Wait Until Page Contains Element        npsp:label:${DATE}[Field Label]
+    Page Should Contain Element             npsp:label:${CAMPAIGN}[Field Label]
+    Page Should Contain Element             npsp:label:${PAY}[Field Label]
+    Page Should Contain Element             npsp:label:${RECORD_TYPE}[Field Label]
+    Fill Gift Entry Form
+    ...                                     Data Import: Donation Donor=Contact1
+    ...                                     Data Import: Contact1 Imported=${data}[contact][Name]
+    ...                                     ${DATE}[Field Label]=Today
+    ...                                     Opportunity: Amount=150
+    ...                                     ${CAMPAIGN}[Field Label]=${CAMPAIGN_REC}[Name]
+    ...                                     ${PAY}[Field Label]=74454354
+    ...                                     ${RECORD_TYPE}[Field Label]=Donation
+    Click Button                            Save & Enter New Gift
+    #Verify gift is created and labels are updated on the table and process batch
+    Verify Gift Count                       1
+    Wait Until Page Contains Element        npsp:gift_entry.field_span:Batch Gifts,${DATE}[Field Label]
+    Page Should Contain Element             npsp:gift_entry.field_span:Batch Gifts,${CAMPAIGN}[Field Label]
+    Page Should Contain Element             npsp:gift_entry.field_span:Batch Gifts,${PAY}[Field Label]
+    Page Should Contain Element             npsp:gift_entry.field_span:Batch Gifts,${RECORD_TYPE}[Field Label]
```

Comment:
Why are you specifically using locator manager here ? Do you intend to update the npsp repo to use the locators here this way instead of creating keywords (for simple actions like this: Page should contain element) ??

Reply:
I am not planning on removing any existing keywords in NPSP repo but thinking may be best to use for simple things like this going forward.